src/gsi/chemmod.f90

```diff
@@ -157,10 +162,10 @@ module chemmod
         'AOLGAJ', 'AISO1J', 'AISO2J', 'AISO3J',   'ATRP1J', 'ATRP2J',&
         'ASQTJ',  'AOLGBJ', 'AORGCJ']
 ! fv3smoke
-  integer(i_kind), parameter ::  naero_smoke_fv3=2 
+  integer(i_kind), parameter ::  naero_smoke_fv3=3 
 
   character(len=max_varname_length), dimension(naero_smoke_fv3), parameter :: &
-        aeronames_smoke_fv3=[character(len=max_varname_length) ::  'smoke','dust' ]
+        aeronames_smoke_fv3=[character(len=max_varname_length) ::  'smoke','dust','coarsepm']
```

Comment:
"coarsepm" is added here. However it is never used. And we have to modify Line 282 of src/gsi/setuppm2_5.f90
 to remove the "coarsepm" contribution

Reply:
Yes, you are correct. This was done by -1 in the do loop. For example, 
do i=2,naero_smoke_fv3-1 ! remove contribution from coarsepm

Reply:
Right, if we will remove coarsepm in src/gsi/setuppm2_5.f90 (line 281), why would we add it here?